server/client_manager.py

```diff
@@ -191,7 +196,7 @@ def send_done(self):
             self.send_command('HP', 1, self.area.hp_def)
             self.send_command('HP', 2, self.area.hp_pro)
             self.send_command('BN', self.area.background)
-            self.send_command('LE', *self.area.get_evidence_list())
+            self.send_command('LE', *self.area.get_evidence_list(self))
```

Comment:
this is obviously a confusing design, stating `self` twice. you probably would like to use a default argument.